R/uvhydrograph-data.R

```diff
@@ -462,13 +477,27 @@ readSecondaryTimeSeriesUvInfo <- function(reportObject) {
 parseCorrectionsAsTable <- function(corrections) {
   if(!is.null(corrections) && nrow(corrections) > 0) {
     corrections_table <- as.data.frame(cbind(seq(nrow(corrections)), as.character(corrections[['time']]), corrections[['comment']]), stringsAsFactors=FALSE)
-    colnames(corrections_table) <- c("", "Time", "Comments")
+    colnames(corrections_table) <- c("", "Time", "Correction Comments")
```

Comment:
renaming this based on task description from laura to differentiate the two comment tables